{
  "deprecated": false,
  "cpeName": "cpe:2.3:a:redhat:ansible_engine:2.8.11:*:*:*:*:*:*:*",
  "cpeNameId": "62B6A7C5-297D-43F6-B53A-14488AD3E27E",
  "created": "2020-05-14T14:17:01.167",
  "lastModified": "2020-05-14T14:17:01.167",
  "titles": [
    {
      "title": "Red Hat Ansible Engine 2.8.11",
      "lang": "en"
    }
  ],
  "refs": [
    {
      "ref": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1746",
      "type": "Advisory"
    }
  ]
}
